Coding/Java/src/org/qcschema/QCSchemaUnits.java

```diff
@@ -0,0 +1,358 @@
+package org.qcschema;
+
+import java.util.Hashtable;
+import java.util.Map;
+
+import java.util.ArrayList;
+
+import org.jmol.viewer.Viewer;
+
+/**
+ * A general Java class for working with QCShema units and array types.
+ * 
+ * j2sNative blocks can be ignored -- they just increase efficiency in the JavaScript rendition of Jmol.
+ *  
+ */
+public class QCSchemaUnits {
+
+  public final static String version = "QCJSON 0-0-0.Jmol_"
+      + Viewer.getJmolVersion().replace(' ', '_');
+
+  // 
+  //  source: http://cccbdb.nist.gov/hartree.asp
```

Comment:
This is quite limiting in terms of accuracy. Please see here for better precision:
https://physics.nist.gov/cuu/Constants/